elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let taboola = window._taboola || [];
```

Comment:
Don't know how taboola works, just guessing. Otherwise yay <3 taboola and it being setup.